### ISSUE TITLE
Upgrade RDS Instance Types 

### ISSUE THREAD

**Issue by ben851** [cds-snc/notification-planning-core#387]
# Description

As an operator of Notify, I would like our quicksight data to stay up to date
So that end users can leverage the features of quicksight
*PLEASE read the acceptance criteria before doing this work. :) 

## WHY are we building?

The quicksight data refresh jobs are failing and crashing the RDS instance that the refresh is running on due to too much data

## WHAT are we building?

Increase RDS instance size as a temporary measure to resolve the issue

## VALUE created by our solution

Quicksight data works again!

# Acceptance Criteria

- [x] Verify if the storage size can be increased without actually doing the work in this card
- [x] Production instance size is increased
- [x] Quicksight data refereshes work

# QA Steps

- [x] Quicksight data refresh works

**Comment by sastels** [comment]
Manual upgrades on staging while doing a 1 email / second soak test:

**Started write upgrade 10:33 staging**

One 500 error (api lambda ssl closing error)
Locust got: HTTPError('500 Server Error: Internal Server Error for url: /v2/notifications/email')

10 “SSL connection has been closed unexpectedly” related celery errors at 10:35 (triggered alarm :/ )
	- 2 process-ses-result jobs failed, leaving 2 notifications in “sent” state
78b009f6-ed85-4ac3-aaf6-3f56e4b503bc
6147850f-be62-4716-ac17-228eef9aa3fa

No failed emails

**Next modify a reader! 10:41**

Done 10:49

No errors

**Last one (writer) 10:53**

Finished 11:00
No errors


TL;DR
- Probably want to do the readers first, then the writer
- One sec of api downtime, 2 process-ses-result crashed

**Comment by sastels** [comment]
Gonna role them all back and see what the impact of just doing it all in terraform would be

**Comment by sastels** [comment]
PR to upgrade:
https://github.com/cds-snc/notification-terraform/pull/1454

**Comment by P0NDER0SA** [comment]
Just updated the branch and we will see where this is tomorrow.
Jimmy would like to discuss when we can push this to production.  Let's discuss this!

**Comment by P0NDER0SA** [comment]
Steve is gonna do a smoke test on staging today and we will wait until next week to push this to production.  Date to be confirmed.

**Comment by sastels** [comment]
upgraded on staging via a terraform PR (setting instance types and `apply_immediately=true`

- all three started modifying at once
- had one 500 error in locust as before
- had 6 `SSL connection has been closed unexpectedly` celery errors but no delivery_receipt crashes this time (so no notifications were stuck in a sending state)

Recommend we do the upgrade with a terraform PR off hours.

**Comment by sastels** [comment]
Will do it tonight at 8 pm EST

**Comment by sastels** [comment]
Doing in prod!
one reader took 6m 56s
next 12m49s 
writer 19m54s

[4 SES processing errors and one Pinpoint processing error](https://ca-central-1.console.aws.amazon.com/cloudwatch/home?region=ca-central-1#logsV2:logs-insights?queryDetail=~(editorString~'fields*20*40timestamp*20as*20Time*2c*20kubernetes.container_name*20as*20Deployment*2c*20log*0a*7c*20filter*20kubernetes.container_name*20like*20*2f*5ecelery*2f*0a*7c*20filter*20*40message*20like*20*2fERROR*5c*2f.*2aWorker*2f*0a*7c*20sort*20*40timestamp*20desc*0a~end~0~source~(~'*2faws*2fcontainerinsights*2fnotification-canada-ca-production-eks-cluster*2fapplication)~start~-10800~timeType~'RELATIVE~unit~'seconds))

**Comment by sastels** [comment]
One note is that our soak test should be more like 3 emails / second, say, rather than 1 email / second to allow us to better anticipate alerts

**Comment by sastels** [comment]
Notifications dataset successfully refreshed last night :tada:

**Comment by sastels** [comment]
Free local memory now drops to 33.4G when doing the dataset refresh and does not crash the refresh :tada:
<img width="1102" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd1lpQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e11a98645fe475d4520e0adacbe361d1a9790a3e/image.png" alt="image.png" />
